shared/profile/user/index.js

```diff
@@ -536,6 +558,9 @@ export const styles = Styles.styleSheetCreate({
       paddingRight: Styles.globalMargins.medium,
     },
   }),
+  proveIt: {
```

Comment:
gap `Box2` above instead

Reply:
that puts gaps between the proofs which we don't want (i tried that first)